### ISSUE TITLE
Travis colorama.ansi doesn't know BLUE

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/bioagents#52]
```
======================================================================
ERROR: run_test (tests.tra_test.TraTestModel2)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/sorgerlab/bioagents/tests/integration.py", line 104, in run_test
    assert self.is_correct_response(), self.give_feedback()
  File "/home/travis/build/sorgerlab/bioagents/tests/integration.py", line 123, in give_feedback
    color_diff(self.expected, self.output.to_string())
  File "/home/travis/build/sorgerlab/bioagents/tests/integration.py", line 43, in color_diff
    Fore.BLUE + sm.b[b0:b1] + Style.RESET_ALL)
AttributeError: type object 'Back' has no attribute 'BLUE'
```
note that this only happens when there is an underlying failure in a test (a string mismatch) but the issue is not related to TraTestModel2 specifically.

**Comment by pagreene** [comment]
Does this happen on your system or on Travis only? I saw something on Travis, and was already trying to figure it out.

**Comment by pagreene** [comment]
I pushed a branch with the fix to sorgerlab and submitted a pull request https://github.com/sorgerlab/bioagents/pull/55

**Comment by bgyori** [comment]
Fixed by #55
